processed 5 tasks

init:
A: object(0,0)

task 1 'publish'. lines 8-15:
created: object(1,0)
mutated: object(0,1)
gas summary: computation_cost: 1000000, storage_cost: 4932400,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2 'programmable'. lines 17-20:
created: object(2,0), object(2,1)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 3420000,  storage_rebate: 0, non_refundable_storage_fee: 0

task 3 'programmable'. lines 22-24:
Error: Transaction Effects Status: Invalid command argument at 1. Invalid argument to private entry function. These functions cannot take arguments from other Move functions
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 1, kind: InvalidArgumentToPrivateEntryFunction }, source: None, command: Some(1) } }

task 4 'programmable'. lines 26-29:
Error: Transaction Effects Status: Invalid command argument at 0. The argument cannot be instantiated from raw bytes
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidUsageOfPureArg }, source: Some("Non-primitive argument at index 0. If it is an object, it must be populated by an object"), command: Some(1) } }
